### PR TITLE
Fixed some clippy warnings

### PR DESCRIPTION
**Pull request by llogiq**
Just some readability nits, but may be useful nonetheless.

### CODE REVIEW

File: src/interpreter.rs [hunk 1]
```diff
@@ -169,8 +169,8 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
 
                 for stmt in &block_data.statements {
                     self.log(0, || print!("{:?}", stmt));
-                    let mir::StatementKind::Assign(ref lvalue, ref rvalue) = stmt.kind;
-                    let result = self.eval_assignment(lvalue, rvalue);
+                    let mir::StatementKind::Assign(ref l_value, ref r_value) = stmt.kind;
+                    let result = self.eval_assignment(l_value, r_value);
```
Comment (eddyb) [inline src/interpreter.rs]: What is even doing this?

**Comment by llogiq** [comment]
clippy decided that `lvalue` and `rvalue` are too similar and that the underscore would aid readability.

File: src/interpreter.rs (hunk 1 above)
Comment (tsion) [inline src/interpreter.rs]: @llogiq Considering these names are only used on two *adjacent* lines, I'd rather stick with the existing names to be consistent with the rest of the code.

**Comment by tsion** [comment]
Apart from the `lvalue`/`rvalue` change I'm ready to merge this.

**Comment by llogiq** [comment]
@tsion Wish granted :smile:

**Comment by tsion** [comment]
@llogiq Thanks!

**Comment by llogiq** [comment]
You're welcome. :+1:

**Pull request merged**